science/giza/Portfile

```diff
@@ -17,12 +16,12 @@ long_description    giza is a 2D scientific plotting library \
                     Builds both libpgplot and libcpgplot that directly replace the PGPLOT libraries.
 
 homepage            https://danieljprice.github.io/giza/
-github.tarball_from releases
+github.tarball_from archive
```

Comment:
This project offers release downloads; release downloads are preferred over archive downloads.

Reply:
Does it actually work for you? I got an error with releases, that is why changed it to archive, which worked fine.

Reply:
@ryandesign I will try again with releases today.

Reply:
Ok, fixed. Needed distname and extract.rename.